app/models/user.rb

```diff
@@ -1,10 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] [OPTIONAL] However, a part of the requirements states that `:confirmable` should be enabled in the user model but you did not do that. With `:confirmable`, users are required to verify their email addresses after signing up. This adds an extra layer of security and ensures that the provided email addresses are valid.
- Kindly implement this fix!